### DISCUSSION TITLE
Does your c2v translator have successfully translated any big project ?

### DISCUSSION THREAD

**Discussion by Jimmy2099** [vlang/c2v#125] [Q&A]
Does your c2v translator have successfully translated any big project ?

I mean big project ,such as linux any version kernal ? gui tools such as blender most of it is c and some c++ 、python binding？

or currently just only work for some show case?

**Comment by ArtemkaKun** [comment]
I'm not sure what do you mean under a "big" project and if DOOM64 game is big or small for you, but right now the biggest project (I assume) that C2V can translate is DOOM64

**Comment by wwlittewayne** [comment]
[OFF-TOPIC] Carbon language seems to share most goals with V. I was wondering which one I should choose when both V and Carbon become  mature.

**Reply by ylluminate** [reply]
This is off-topic. Thanks. Go to [Discord](https://discord.gg/vlang) → #languages, #programming, or #random-and-offtopic.

**Comment by ArtemkaKun** [comment]
I think you should address such questions on Discord channel rather than here.

Up to me - Carbon and V has too different syntax, and also Carbon is OOP language (afaik), while V is not OOP.
